.gitignore

```diff
@@ -1,5 +1,6 @@
 node_modules/
 .DS_Store/
+help.txt
```

Comment:
if help.txt contains useful information, then please do not ignore it. Perhaps you need to create `docs` folder?

Reply:
Deleted.